code401challenges/src/main/java/code401challenges/stacksandqueues/Queue.java

```diff
@@ -0,0 +1,75 @@
+package code401challenges.stacksandqueues;
+
+import java.util.NoSuchElementException;
+
+public class Queue {
+    // This object should be aware of a default empty value assigned to front when the queue is created.
+
+    public Node front;
+    public Node back;
+
+    // this is a constructor
+    public Queue(){
+        front = null;
+    }
+
+    //Define a method called isEmpty that does not take an
+    // argument, and returns a boolean indicating whether
+    // or not the queue is empty.
+
+    public boolean isEmpty(){
+        return front == null;
+    }
+
+    //Define a method called enqueue which takes any value as
+    // an argument and adds a new node with that value to the
+    // back of the queue with an O(1) Time performance.
+
+    public void enqueue(int value){
+        Node runner =  new Node(value);
```

Comment:
I recommend renaming this from `runner` to something like `n` or `temp` or `newNode`, since it is not being using to iterate through the Queue - it's just the new node being inserted into the Queue.

Reply:
I will thank you